src/libm/sleefsimdsp.c

```diff
@@ -412,6 +382,63 @@ EXPORT CONST vfloat xcosf_u1(vfloat d) {
   return u;
 }
 
+#ifndef ENABLE_GNUABI
```

Comment:
Why not change the prototype when enabling GNUABI instead of removing the function?